controls/datagrid/columns/header.md

```diff
@@ -32,40 +32,44 @@ To customize text inside the header you have to use the `HeaderText` property. T
 
 ## Sorting
 
-You can easily sort a particular column tapping ot its header. When the data is sorted by a column, its header changes its appearance and shows the sort direction via an indicator.
+The user of the application can sort a particular column when tapping on its header. When the data is sorted by a column, the sort indicator shows in the header.
 
 ![.NET MAUI DataGrid Column Header Sorting indicator](../images/column-header-sorting.png)
 
 To learn more about the sorting functionality take a look at the [Sorting]({%slug datagrid-sorting-overview%}) article.
 
 ## Filtering
 
-The header of the column hosts the built-in filtering mechanism (the filter indicator which opens the Filtering UI), which allows you to filter the data by the columns' values.
+The header of the column hosts the built-in filtering mechanism (the filter indicator which opens the Filtering UI), which allows the user to filter the data by the columns' values.
 
 ![.NET MAUI DataGrid Column Header filter indicator](../images/column-header-filtering.png)
 
 To learn more about the filtering functionality take a look at the [Filtering]({%slug datagrid-filtering-overview%}) article.
 
 ## Styling 
 
-Use the `HeaderStyle` property in order to style the DataGridColumn header.
+Use the `HeaderStyle` property to style the `DataGridColumn` header.
 
 Check the [.NET MAUI DataGrid Column Header Styling]({%slug datagrid-columns-styling%}#headerstyle) topic for more information about the styling options you can use. 
 
 ## Header Content Customization
 
-You can easily customize the content of the Header using the `HeaderContentTemplate`(`DataTemplate`) property.
+You can customize the content of the Header using the `HeaderContentTemplate`(`DataTemplate`) property.
 
 Define the `DataTemplate` for the header:
 
 <snippet id='datagrid-headercontenttemplate-datatemplate' />
 
-Define the `HeaderContentTemplate` in the DataGridColumn:
+Define the `HeaderContentTemplate` in the DataGrid column:
 
 <snippet id='datagrid-headerfootercontenttemplate' />
 
 ![.NET MAUI DataGrid Column Header Template](../images/header-content-template.png)
 
+## Fully Customize the Column
+
+Customize the Column Header using the `HeaderContentTemplate` to achieve the desired full customization of the column.
```

Comment:
Update this sentence and add more details of the customization. Maybe we do not need 2 sections for this, only one is enough. provide example or update the existing example in the SDK app.